libs/gru-http/gru-http.gradle

```diff
@@ -21,4 +21,9 @@ dependencies {
     api "com.squareup.okhttp3:okhttp:${okHttpVersion}"
 
     compileOnly "org.codehaus.groovy:groovy:$groovyVersion"
+
+    testImplementation "com.stehno.ersatz:ersatz:$ersatzVersion"
+    testImplementation 'javax.activation:activation:1.1.1'
+    testImplementation "io.undertow:undertow-core:$undertowVersion"
+    testImplementation 'org.slf4j:slf4j-nop:2.0.5'
```

Comment:
just in case - add a real implementation of Slf4J

Reply:
I changed it to Log4j as requested, but I think I should give more context to this dependency. My intention was not to use any logging implementation but just fix `java.lang.NoClassDefFoundError: org.slf4j.Logger` caused by adding Ersatz. I thought using `slf4j-nop` is OK, especially since no logging framework was used before in tests, and I didn't want to introduce any. The idea was to leave this decision to the future selves, whenever we see that the logging framework is useful and we want to pick up a specific one :+1:

Reply:
I'm only afraid that the no-op logging can hide some important warning/error messages in the future